tests/regression/09-regions/23-evilcollapse_rc.c

```diff
@@ -40,15 +42,15 @@ static inline void list_add(struct list_head *new, struct list_head *head) {
 }
 
 inline static struct list_head *lookup (int d) {
-  int hvalue;
+  int hvalue = __VERIFIER_nondet_int();
   struct list_head *p;
   p = c.slot[hvalue].next;
```

Comment:
I guess if you are strict about it, such things are possibly undefined behavior, as `hvalue` might go beyond the array.
Now, we don't care about it, but I don't know if the SV-Comp folks are also so relaxed about UB.